src/Header/ContentType.php

```diff
@@ -66,6 +74,13 @@ public function getFieldValue($format = HeaderInterface::FORMAT_RAW)
 
         $values = [$prepared];
         foreach ($this->parameters as $attribute => $value) {
+            if (HeaderInterface::FORMAT_ENCODED === $format &&
+                !Mime::isPrintable($value)) {
```

Comment:
Push the `&&` to the second line, and the closing `)` and opening `{` onto a line by themselves.